packages/lib/src/components/Footer/Footer.module.css

```
@@ -17,15 +17,14 @@
  * along with Essencium Frontend. If not, see <http://www.gnu.org/licenses/>.
  */
 
-.footer__app-shell {
-  padding: var(--mantine-spacing-md);
-}
+
 
 .footer__flex {
-  margin-left: var(--mantine-spacing-xs);
+  padding: 0 var(--mantine-spacing-xs);
```

My last annotation, I tried it with more horizontal padding and it looks a bit spacier 🙂 

```suggestion
  padding: 0 var(--mantine-spacing-xl);
```